test/test-supervisor.js

```diff
@@ -55,130 +58,154 @@ describe('supervisor', function(done) {
     }
   }
 
-  function run(dir, args, expectations) {
-    it('should run [' + args.join(',') + '] from ' + dir, function(done) {
-      runSlr(dir, args, expectations, done);
+  function run(t, dir, args, expectations) {
+    t.test('should run [' + args.join(',') + '] from ' + dir, function(t) {
+      runSlr(dir, args, expectations, function() {
+        afterEach(end);
```

Comment:
Methods on `t` are pre-bound to `t` specifically for simplifying code like this :-)

```js
t.test('should run [' + args.join(',') + '] from ' + dir, function(t) {
  runSlr(dir, args, expectations, function() {
    afterEach(t.end);
  });
});
```